### PR TITLE
Bhr 33 - Create enpoints for books

### PR DESCRIPTION
**Pull request by AlejandroHerr**

### CODE REVIEW

**Comment by AlejandroHerr** [comment]
The branch is called bhr-32 but it was a mistake of mine since I understood that 'POST complete book data to store it on the database' referred to the endpoint.

File: backend/src/book/router.spec.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+};
+
+const app = express();
+app.use('/', router);
+app.db = db;
+
+describe('Book router', () => {
+  beforeEach(() => db.sequelize.sync({ force: true })
+    .then(() => db.models.User.create(rawUser)));
+  after(() => db.sequelize.drop());
```
Comment (AlejandroHerr) [inline backend/src/book/router.spec.js]: ```sync()``` with ```{force: true}``` drops the tables before sync'ing the models. I was just writing the before for some safety.

Comment (caderek) [inline backend/src/book/router.spec.js]: ok

File: backend/src/book/router.spec.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    it('should respond with Not Acceptable if the client does not accept json', () => {
+      request(app)
+        .get('/books')
+        .set('Accept', 'application/xml')
+        .expect('Content-Type', /json/)
+        .expect(HTTPStatus.NOT_ACCEPTABLE);
+    });
+  });
+  describe('POST /book', () => {
+    it('should create a new Book and respond with No Content', () => {
```
Comment (AlejandroHerr) [inline backend/src/book/router.spec.js]: I added the Location header.

**Pull request merged**